### PR TITLE
chore: New hero section on landing page

### LINKED ISSUE #241: New hero section on landing page
### Feature Description

Implement new Hero section.

Figma: [here](https://www.figma.com/file/jGpo62JSykdqBAKYQ8iiDg/Classroomio-Web?type=design&node-id=5112-23873&mode=design&t=Oznrmvc4cVKtIjRk-0)

### Use Case

_No response_

### Proposed Solution

_No response_

### Alternatives (optional)

_No response_

### Additional Context

_No response_

### Please check the boxes that apply to this feature request.

- [ ] I have searched the existing feature requests to make sure this is not a duplicate.
- [ ] I have provided a detailed description of the requested feature.
- [ ] I have explained the use case or scenario for this feature.
- [ ] I have included any relevant technical details or design suggestions.
- [ ] I understand that this is a suggestion and that there is no guarantee of implementation.

### PR DESCRIPTION
**Pull request by tunny17**
## What does this PR do?

Fixes #241 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [x] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [ ] This change requires a documentation update

## How should this be tested?

- Open the landing page (classroomio.com)

### Required

- [x] Filled out the "How to test" section in this PR
- [x] Self-reviewed my own code
- [ ] Commented on my code in hard-to-understand bits
- [ ] Ran `pnpm build`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues

### Appreciated

- [x] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the ClassroomIO Docs if changes were necessary
Screenshot (703)

### CODE REVIEW

File: apps/classroomio-com/src/app.css [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
```
Comment (rotimi-best) [inline apps/classroomio-com/src/app.css]: why are we importing the inter css

File: apps/classroomio-com/src/lib/Hero/Hero.svelte [hunk 2]
```diff
@@ -28,43 +28,109 @@
...
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <rect width="10.9223" height="10.9223" fill="#3ADFEC" />
+      </svg>
+    </div>
+
+    <div class="relative">
+      <a
+        href="https://github.com/rotimi-best/classroomio"
```
Comment (rotimi-best) [inline apps/classroomio-com/src/lib/Hero/Hero.svelte]: Let's change it to `/github`

File: apps/classroomio-com/src/lib/Hero/Hero.svelte [hunk 3]
```diff
@@ -28,43 +28,109 @@
   });
 </script>
 
-<div class="relative mt-[30%] border-b-2 text-center md:mt-36">
+<div class="relative pt-[30%] border-b-2 text-center md:pt-36 bg-[#F5F8FE]">
   <div class="mb-10 flex w-full items-center justify-center">
     <a
       target="_blank"
       rel="noopener noreferrer nofollow"
       href="https://github.com/rotimi-best/classroomio"
```
Comment (rotimi-best) [inline apps/classroomio-com/src/lib/Hero/Hero.svelte]: Change it to `/github`

**Review by rotimi-best — CHANGES_REQUESTED** [review]

File: apps/classroomio-com/src/app.css (hunk 1 above)
Comment (tunjiadeyemi) [reply apps/classroomio-com/src/app.css]: Usman changed the font of the hero section to inter so i had to import it and make it a class to reuse

File: apps/classroomio-com/src/lib/Hero/Hero.svelte (hunk 2 above)
Comment (tunjiadeyemi) [reply apps/classroomio-com/src/lib/Hero/Hero.svelte]: alright

File: apps/classroomio-com/src/lib/Hero/Hero.svelte (hunk 3 above)
Comment (tunjiadeyemi) [reply apps/classroomio-com/src/lib/Hero/Hero.svelte]: alright

File: apps/classroomio-com/src/app.css (hunk 1 above)
Comment (rotimi-best) [reply apps/classroomio-com/src/app.css]: let's use the font that we used to have. No need for inter

Comment (tunjiadeyemi) [reply apps/classroomio-com/src/app.css]: alright, noted

**Review by rotimi-best — APPROVED** [review]

**Pull request merged**